process/block/metablock.go

```diff
@@ -2615,6 +2615,18 @@ func (mp *metaProcessor) MarshalizedDataToBroadcast(
 
 	mrsData := make(map[uint32][]byte, len(bodies))
 	for shardId, subsetBlockBody := range bodies {
+
+		if hdr.IsStartOfEpochBlock() {
+			log.Debug("metaProcessor.MarshalizedDataToBroadcast",
+				"len subsetBlockBody", len(subsetBlockBody),
+			)
+			for _, mb := range subsetBlockBody {
+				log.Debug("subsetBlockBody",
+					"mb type", mb.Type,
+				)
+			}
+		}
```

Comment:
So only for logging, correct? Then, it can also be a separate function. Can also stay as it is.